components/colorwheel/index.css

```diff
@@ -1,27 +1,68 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
 .spectrum-ColorWheel {
-  /* todo: fix this in DNA */
-  --spectrum-colorwheel-focus-ring-size: var(--spectrum-global-dimension-static-size-25);
-  --spectrum-colorwheel-border-radius: 100%;
-  --spectrum-colorwheel-width: calc(var(--spectrum-global-dimension-size-125) * 16);
   --spectrum-colorwheel-height: var(--spectrum-colorwheel-width);
+  --spectrum-colorwheel-border-color: var(--spectrum-transparent-black-200);
+  --spectrum-colorwheel-border-width: var(--spectrum-border-width-100);
+  --spectrum-colorwheel-fill-color-disabled: var(--spectrum-disabled-background-color);
+  --spectrum-colorwheel-track-width: var(--spectrum-color-control-track-width);
+  --spectrum-colorwheel-colorarea-margin: var(--spectrum-color-wheel-color-area-margin);
+  --spectrum-colorhandle-size: 16px; /* TODO replace with new token for ColorHandle, placeholder for old token var(--spectrum-colorhandle-size) */
```

Comment:
does the color handle change size? I think so - at least that's what we did in SVG when we created the handle.
You might want to move it into the `.specturm-medium / large area`
<img width="1924" alt="Screen Shot 2023-02-21 at 11 06 46" src="https://user-images.githubusercontent.com/52184321/220436470-48593613-5fdd-41cf-8245-e076c732e52b.png">

Reply:
good call and yes it does! Got this fixed.